### ISSUE TITLE
Why not perform the transfer learning setting?

### ISSUE THREAD

**Issue by ha-lins** [illidanlab/MoCL-DK#5]
Hi @mengyings,

Thanks for the great work! I have a minor question as the title shows. You say: `We do not evaluate transfer learning setting in
this paper where a pretrained model is applied to another dataset`. The transfer learning setting could be more challenging and practical. Following Hu et. al, you can pretrain on the unlabeled `ZINC15` database, and then evaluated on these `downstream tasks`. Could it be possible to perform this setting for MoCL-DK? Thanks!

Reference:
[1] Strategies to pre-train graph neural networks. Hu et al. 2020.

**Comment by mengyings** [comment]
Yes, we are currently pretraining on the ZINC dataset and will release the pre-trained model once done.
